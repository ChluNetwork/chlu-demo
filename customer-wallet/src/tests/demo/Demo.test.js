import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Demo from 'containers/Demonstrator/Demo'

describe('<Demo />', () => {

  it('should render correctly demo container', () => {
    const wrapper = shallow(
      <Demo />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
