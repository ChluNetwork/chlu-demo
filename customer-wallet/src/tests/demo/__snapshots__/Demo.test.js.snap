// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Demo /> should render correctly demo container 1`] = `
<div
  className="demo"
>
  <div
    className="wallets-container"
  >
    <div
      className="wallets-container__item"
    >
      <Connect(WithFxRates) />
    </div>
    <div
      className="wallets-container__item"
    >
      <Connect(WithFxRates) />
    </div>
  </div>
  <div
    className="demo-logging"
  >
    DEMO DEMO DEMO
  </div>
</div>
`;
